tests/test_strategies.py

```diff
@@ -198,6 +210,25 @@ def test_ridiculously_parallel_tree_search(self):
             player.tree_search(num_parallel=50)
         self.assertNoPendingVirtualLosses(player.root)
 
+    def test_long_game_tree_search(self):
+        player = MCTSPlayerMixin(DummyNet())
+        endgame = go.Position(
+            board=TT_FTW_BOARD,
+            n=MAX_DEPTH-2,
+            komi=2.5,
+            ko=None,
+            recent=(go.PlayerMove(go.BLACK, (0, 1)),
+                    go.PlayerMove(go.WHITE, (0, 8))),
+            to_play=go.BLACK
+        )
+        player.initialize_game(endgame)
+
+        # Test that an almost complete game
+        for i in range(100):
```

Comment:
bump this down? I noticed that lots of tree searching in tests can take like a second to run (a whole second!). It should only take like 10*8 searches to figure this test out.

Reply:
That won't actually work to test the behavior, unfortunately.  With all the empty spots in the new test, and an equal policy for all points, it literally tries every permutation, so even at only N-2 moves out, we won't actually ground out.  I will rework the board position though standby

Reply:
Done